WORKING_GROUPS.md

```diff
@@ -272,4 +230,228 @@ The [Node.js Code of Conduct][] applies to this WG.
 [Node.js Moderation Policy]: https://github.com/nodejs/TSC/blob/master/Moderation-Policy.md
 ```
 
-[Community Committee]: https://github.com/nodejs/community-committee
\ No newline at end of file
+[Community Committee]: https://github.com/nodejs/community-committee
+
+## Current Working Groups
+
+* [Website](#website)
+* [Streams](#streams)
+* [Build](#build)
+* [Diagnostics](#diagnostics)
+* [i18n](#i18n)
+* [Docker](#docker)
+* [Addon API](#addon-api)
+* [Benchmarking](#benchmarking)
+* [Post-mortem](#post-mortem)
+* [Intl](#intl)
```

Comment:
`Evangelism` is missing in the list (it should be before `Docker`) and they are not in order.
`Intl` should come after `i18n`.

Reply:
The `Evangelism` Working Group has been transferred to the Community Committees scope of responsibility.

Reply:
@BridgeAR ... if it's OK, then I'd like to reorder those in a separate PR

Reply:
Alright but it is described down below and the titel says "Current Working Groups". Therefore I think the working group should either be listed here or the detailed information about it should be removed, don't you think so?

Reply:
Sure, I do not think it should hold the PR from being merged.

Reply:
Yep! I missed that it was still mentioned!